src/components/flowchart/components/sliced-pipeline-action-bar/sliced-pipeline-action-bar.js

```diff
@@ -1,27 +1,83 @@
 import React from 'react';
+import classnames from 'classnames';
 import Button from '../../../ui/button';
+import CommandCopier from '../../../ui/command-copier/command-copier';
+import CutIcon from '../../../icons/cut';
+import IconButton from '../../../ui/icon-button';
+import { sidebarWidth, metaSidebarWidth } from '../../../../config';
 
 import './sliced-pipeline-action-bar.scss';
 
-export const SlicedPipelineActionBar = ({
-  chartSize,
-  slicedPipeline,
-  visibleSidebar,
-}) => {
-  const { outerWidth } = chartSize;
+export const SlicedPipelineActionBar = React.forwardRef((props, ref) => {
+  const {
+    chartSize,
+    displayMetadataPanel,
+    isSlicingPipelineApplied,
+    onApplySlicingPipeline,
+    onResetSlicingPipeline,
+    runCommand,
+    slicedPipeline,
+    visibleSidebar,
+  } = props;
+  const { outerWidth: screenWidth } = chartSize;
+  const buffer = 200;
```

Comment:
may be we can have these values in constant or config files to manage like SLICED_PIPELINE_BUFFER_WIDTH ? (not sure of the naming)

Reply:
I could give it a better name rather just buffer? Im not sure if we should move it to config as it won't be shared between any other components 🤔

Reply:
yeah makes sense to name it better. `buffer` did feel generic.